shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -931,9 +943,9 @@ void collectRoutes(List<SemanticsObject> edges) {
             if (hasFlag(Flag.SCOPES_ROUTE)) {
                 edges.add(this);
             }
-            if (children != null) {
-                for (int i = 0; i < children.size(); ++i) {
-                    children.get(i).collectRoutes(edges);
+            if (childrenInTraversalOrder != null) {
```

Comment:
@jonahwilliams To determine the route, we want to go in traversal order, right?

Reply:
It is possible I had this backwards, but what I wanted is whatever "Semantic order" is